wherehows-web/mirage/factories/dataset.ts

```diff
@@ -40,5 +40,11 @@ export default Factory.extend({
     schema(id: number) {
       return id === 0 ? testSchemaA : 'abcd';
     }
+  }),
+
+  withHealth: trait({
```

Comment:
I think this should be a separate factory, so you can test datasets vs dataset-health independently, this might make testing brittle since it introduces coupling between separate features